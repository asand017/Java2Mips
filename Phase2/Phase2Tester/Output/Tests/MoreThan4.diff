--- ./Phase2Tester/SelfTestCases/MoreThan4.out	2011-04-28 14:55:20.000000000 -0700
+++ ./Phase2Tester/Output/Tests/MoreThan4.out	2017-05-21 21:54:36.647001739 -0700
@@ -1,13 +1,4 @@
-1
-2
-3
-4
-5
-6
-6
-5
-4
-3
-2
-1
-0
+--- Error (ProgramFault) ---
+expecting argument 1 to be an integer, found undefined value
+  Main() line 13
+  MT4.Start(<Heap.0>+0 6) line 19
