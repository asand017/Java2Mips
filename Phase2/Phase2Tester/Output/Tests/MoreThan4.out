--- Error (ProgramFault) ---
expecting argument 1 to be an integer, found undefined value
  Main() line 13
  MT4.Start(<Heap.0>+0 6) line 18
